### DISCUSSION TITLE
Version 0.3.0 released

### DISCUSSION THREAD

**Discussion by mjstrasser** [klogging/klogging#45] [Announcements]
Version 0.3.0 of Klogging has been released with updates, including:

- [Configuration using JSON files](https://klogging.io/docs/configuration/json) on the classpath or specified by an environment variable.
- Support for using Klogging from Java, including the [Klogging SLF4J binding library](https://github.com/klogging/slf4j-klogging) that is in early development.
- Interpretation of message templates by the `RENDER_SIMPLE` event renderer.

**Comment by littledot** [comment]
I came across this library while looking for logging libs. Just curious why you make special `suspend` version of the logging functions like info, warn, etc. 

Thanks for your work.

**Comment by mjstrasser** [comment]
HI @littledot, thanks for your interest in Klogging!

To answer your question, `suspend` versions of logging functions allow the inclusion of information from the coroutine context in log events. This feature is described in [Contextual event items](https://klogging.io/docs/coroutines/context) in the documentation. Let me know if you have other questions.
